content/privacy-policy.md

```diff
@@ -0,0 +1,139 @@
+---
+html: privacy-policy.html
+parent: xrp-ledger-overview.html
+blurb: 
+subtitle: Privacy Policy
+top_nav_grouping: About
+sidebar: disabled
+template: page-privacy-policy.html.jinja
+# filters:
+#   - faq
+name: Privacy Policy
+---
+# XRPL.org Privacy Policy
+
+Last updated: August 23, 2021
```

Comment:
This date seems wrong since it's already 2023 and this is the first time we've actually put the policy on the website…

Reply:
Fixed